### PR TITLE
openresty: 1.9.3.1 -> 1.13.6.1

### PR DESCRIPTION
**Pull request by lblasc**
###### Motivation for this change

Bump, cc @thoughtpolice 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg build openresty

File: pkgs/servers/http/openresty/default.nix [hunk 1]
```diff
@@ -1,25 +1,32 @@
-{ stdenv, fetchurl, fetchgit, openssl, zlib, pcre, libxml2, libxslt, gd, geoip
-, perl }:
+{ stdenv, fetchurl, fetchgit, openssl, zlib, pcre, postgresql, libxml2, libxslt,
+gd, geoip, perl }:
 
 assert stdenv.isLinux;
```
Comment (Mic92) [inline pkgs/servers/http/openresty/default.nix]: This line can be removed. `platforms` is enough.

Comment (lblasc) [reply pkgs/servers/http/openresty/default.nix]: Thanks for review. Removed.

**Comment by lblasc** [comment]
Package sold also build on darwin, I've removed linux restriction.

**Comment by fpletz** [comment]
@GrahamcOfBorg build openresty

File: pkgs/servers/http/openresty/default.nix [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
   src = fetchurl {
-    url = "http://openresty.org/download/ngx_openresty-${version}.tar.gz";
-    sha256 = "1fw8yxjndf5gsk44l4bsixm270fxv7f5cdiwzq9ps6j3hhgx5kyv";
+    url = "http://openresty.org/download/openresty-${version}.tar.gz";
+    sha256 = "0sv93nw1j597mdpnibmgaa3nnghdk0z6k27vavm8w2c1z9n6w96i";
   };
 
-  buildInputs = [ openssl zlib pcre libxml2 libxslt gd geoip perl ];
+  nativeBuildInputs = [ openssl zlib pcre libxml2 libxslt gd geoip postgresql ];
+  buildInputs = [ perl ];
```
Comment (Mic92) [inline pkgs/servers/http/openresty/default.nix]: Is it possible that you mixed up `nativeBuildInputs` and `buildInputs`. Libraries should be be in `buildInputs` and native tools (the architecture of the host compiling the package) should be in `nativeBuildInputs`.

**Comment by Mic92** [comment]
@GrahamcOfBorg build openresty

**Comment by Mic92** [comment]
Here we go!

File: pkgs/servers/http/openresty/default.nix (hunk 2 above)
Comment (lblasc) [reply pkgs/servers/http/openresty/default.nix]: You are right, `nativeBuildInputs` should be in `buildInputs`, they are needed also in runtime. Thanks for fixup.

**Pull request merged**